docs/postgres.md

```diff
@@ -428,7 +468,18 @@ Now, if we navigate to the GCS bucket, we are going to see backed up data has be
 
 ## Restore PostgreSQL
 
-We are going to restore the database from the backup we have taken in the previous section. We are going to deploy a new database and initialize it from the backup.
+Now, we are going to restore the database from the backup we have taken in the previous section. We are going to deploy a new database and initialize it from the backup.
+
+**Stop Taking Backup of the Old Database:**
```

Comment:
I think we should also say if this was created by AutoBackup, that annotation should be removed.

Reply:
done